src/coreimage.cs

```diff
@@ -1927,6 +1986,32 @@ interface CIImage : NSSecureCoding, NSCopying {
 		[return: NullAllowed]
 		CIImage FromPortraitEffectsMatte (AVPortraitEffectsMatte matte);
 
+		// CIImage_AVSemanticSegmentationMatte
+
+		[TV (13,0), iOS (13,0), Mac (10,15)]
+		[NullAllowed, Export ("semanticSegmentationMatte")]
+		AVSemanticSegmentationMatte SemanticSegmentationMatte { get; }
+
+		[TV (13,0), iOS (13,0), Mac (10,15)]
+		[Export ("initWithSemanticSegmentationMatte:options:")]
+		IntPtr Constructor (AVSemanticSegmentationMatte matte, [NullAllowed] NSDictionary options);
```

Comment:
Do we know about the options? Could it be a string dictionary?

Reply:
string ? no, strong ? maybe :)

It was not documented before but now that 
`typedef NSString *CIImageOption;`
was added we can be pretty sure it's the same as the one I mentioned above

However there's a lot of older members that also use `NSDictionary` so, to keep this PR from calling unicorns, I'll fix that in a separate PR.